From 085277174edbabc4ea00319cfb5692fc7ae09ab6 Mon Sep 17 00:00:00 2001
From: Thomas Ackermann <th.acker@arcor.de>
Date: Wed, 6 Feb 2013 19:44:50 +0100
Subject: [PATCH] Add readme.txt and install.txt to ./Documentation/Makefile

---
 Documentation/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/Makefile b/Documentation/Makefile
index e7375ce..7dc312b 100644
--- a/Documentation/Makefile
+++ b/Documentation/Makefile
@@ -39,6 +39,8 @@ DOC_HTML=$(MAN_HTML) $(OBSOLETE_HTML)
 
 ARTICLES = howto-index
 ARTICLES += everyday
+ARTICLES += readme
+ARTICLES += install
 ARTICLES += git-tools
 ARTICLES += git-bisect-lk2009
 # with their own formatting rules.
-- 
1.8.1.msysgit.1

