From b2685f144b88efa7b263bd3b1810dc001280478c Mon Sep 17 00:00:00 2001
From: Thomas Ackermann <th.acker@arcor.de>
Date: Sat, 29 Sep 2012 10:30:00 +0200
Subject: [PATCH] Disable automatic reconfiguration due to GIT-VERSION

Signed-off-by: Thomas Ackermann <th.acker@arcor.de>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 0f1261b..63e5628 100644
--- a/Makefile
+++ b/Makefile
@@ -1786,7 +1786,7 @@ $(patsubst %.py,%,$(SCRIPT_PYTHON)): % : unimplemented.sh
 	mv $@+ $@
 endif # NO_PYTHON
 
-configure: configure.ac GIT-VERSION-FILE
+configure: configure.ac
 	$(QUIET_GEN)$(RM) $@ $<+ && \
 	sed -e 's/@@GIT_VERSION@@/$(GIT_VERSION)/g' \
 	    $< > $<+ && \
-- 
1.8.1.msysgit.1

